rhscl-postgresql-apb/README.md

```diff
@@ -0,0 +1,25 @@
+rhscl-postgresql-apb
+======================
+
+An apb for deploying [PostgreSQL](https://www.postgresql.org).
+
+## What it does
+* Deploys 1 postgresql container.
+
+## Requirements
+
+## Parameters
+* NAMESPACE: Optional, default 'postgresql-demo-apb', Namespace to deploy the cluster in.
+* POSTGRESQL_DATABASE, default 'admin', Postgresql database name.
+* POSTGRESQL_PASSWORD, default is a randomly generated string, Postgresql database password.
```

Comment:
I think what @fabianvf is getting at is to remove the "lookup('env'..." stuff in the default.yaml and just set defaults where appropriate. Then update the README to suggest the lowercase var names that ansible would expect and pass them in with --extra-vars '{...}' at the end of the command line (and in the more usual case as the broker already does), which I think is reasonable...

@fabianvf is my understanding correct?